src/buildConfig.js

```diff
@@ -106,6 +111,18 @@ function getModulePath(basedir, lookup) {
   )
 }
 
+function findIgnorePatterns(configDir) {
+  const ignoreFilePath = path.resolve(configDir, ".stylelintignore")
+
+  if (!fs.existsSync(ignoreFilePath)) {
```

Comment:
I think we should haven't to make two `fs` calls to see if it exists and then read it. Instead, you can make one call and handle the error if it doesn't exist, handle the content if it does.